Description: disable NEON instructions for architecture armel
Author: Alexander Pozdnyakov <almipo@mail.ru>
Last-Update: 2023-10-07

--- tesseract-5.2.0.orig/configure.ac
+++ tesseract-5.2.0/configure.ac
@@ -170,6 +170,11 @@
 
     ;;
 
+  arm)
+
+    AC_MSG_WARN([No compiler options for $host_cpu])
+    ;;
+    
   aarch64*|arm64)
 
     # ARMv8 always has NEON and does not need special compiler flags.
